lib/src/main/java/com/auth0/android/lock/views/SocialView.java

```diff
@@ -86,8 +87,8 @@ private void init(boolean smallButtons) {
     }
 
     @Override
-    public void onConnectionClicked(String connectionName) {
-        lockWidget.onSocialLogin(new SocialConnectionEvent(connectionName));
+    public void onAuthenticationRequest(@NonNull String strategy,@NonNull String connection) {
+        lockWidget.onSocialLogin(new SocialConnectionEvent(strategy, connection));
```

Comment:
this is just a comment, but maybe the name of this method should better reflect what it does? now it's clear that an authentication has been requested, but the `onSocialLogin` is not that clear (`onSocialLoginRequest`?).